pyhgvs/__init__.py

```diff
@@ -1334,19 +1337,24 @@ def hgvs_normalize_variant(chrom, offset, ref, alt, genome, transcript=None):
 
 def parse_hgvs_name(hgvs_name, genome, transcript=None,
                     get_transcript=lambda name: None,
-                    flank_length=30, normalize=True):
+                    flank_length=30, normalize=True, lazy=False):
     """
     Parse an HGVS name into (chrom, start, end, ref, alt)
 
     hgvs_name: HGVS name to parse.
     genome: pygr compatible genome object.
     transcript: Transcript corresponding to HGVS name.
     normalize: If True, normalize allele according to VCF standard.
+    lazy: If True, discard version information from incoming transcript/gene.
```

Comment:
I can see that this is convenient. So it SGTM. However, the user should be very careful though, because sequence can change quite a bit between transcript versions, especially indel changes.